src/scripts/apis/index.ts

```diff
@@ -0,0 +1,41 @@
+// Types
+import {ServerError, UnauthorizedError} from 'src/types/error'
+
+// Constants
+import {CLOUD} from 'src/shared/constants'
+
+export interface Scripts {
+  scripts?: Script[]
+}
+export interface Script {
+  readonly id?: string
+  name: string
+  description?: string
+  orgID: string
+  script: string
+  language?: ScriptLanguage
+  url?: string
+  readonly createdAt?: string
+  readonly updatedAt?: string
+  labels?: string[]
+}
+export type ScriptLanguage = 'flux' | 'sql'
+
+let getScripts
+
+if (CLOUD) {
+  getScripts = require('src/client/scriptsRoutes').getScripts
```

Comment:
since `scriptsRoutes.ts` file will not be generated until UI runs, it's necessary to add `CLOUD` check